getting-started.md

```diff
@@ -35,7 +31,10 @@ As you can see, this is a very simple _Todo app_. The application uses the most
 
 ### Credential stuffing attack
 For this example, we will only consider the simplest credential stuffing attack. The attack will consist of consecutive login attempts executed one after another. For simulating this scenario, we will use Postman or rather its headless version Newman.
-Go to test scenarios directory:
+
+More specifically, we will use a Nixer fork of Newman with custom modification that provides programmatic access to Cookies - we needed it to automate the test scenarios. You can review our modifications [here on branch feature/cookies-domain-whitelist](https://github.com/nixer-io/newman.git#feature/cookies-domain-whitelist). 
```

Comment:
This link does not actually show our changes to Newman, the following one works better:
`https://github.com/nixer-io/newman/compare/develop...feature/cookies-domain-whitelist`

I would also mention here how exactly this fork is used in our tests - that the github branch is simply referred from the `package.json`.  It might be not obvious, and also such information adds more transparency.

Reply:
That's ok, I don't want to explain everything here, I don't want to distract too much from the main goal of the tutorial. We will add this information in separate section for testing on in testing repo once we will get there.